pyrasterframes/src/main/python/docs/raster-write.pymd

```diff
@@ -39,18 +41,61 @@ display(tile) # IPython.display function
 Within an IPython or Jupyter interpreter, a Spark and Pandas DataFrames containing a column of _tiles_ will be rendered as the samples discussed above. Simply import the `rf_ipython` submodule to enable enhanced HTML rendering of these DataFrame types.
 
 ```python to_samples, evaluate=True
-import pyrasterframes.rf_ipython
 
 samples = spark_df \
     .select(
         rf_extent('proj_raster').alias('extent'),
         rf_tile('proj_raster').alias('tile'),
     )\
-    .select('extent.*', 'tile') \
+    .select('tile', 'extent.*') \
     .limit(3)
 samples
 ```
 
+## Rendering Samples with Color
+
+By default the IPython visualizations use the Viridis color map for each single channel tile. There are other options for reasoning about how color should be applied in the results.
+
+
+### Color Composites
+
+Rendering three different bands of imagery together is called a _color composite_. The bands selected are mapped to the red, green, and blue channels of the resulting display. If the bands chosen are red, green, and blue, the composite is called a true-color composite. Otherwise it is a false-color composite.
+
+Using the @ref:[`rf_rgb_composite`](reference.md#rf-rgb-composite) function, we will compute a three band PNG image as a `bytearray`. The resulting `bytearray` will be displayed as an image in either a Spark or pandas DataFrame display if `rf_ipython` has been imported.
+
+```python, color-composite
+# Select red, green, and blue, respectively
+composite_df = spark.read.raster([[scene(1), scene(4), scene(3)]],
+                                 tile_dimensions=(256, 256))
+composite_df = composite_df.withColumn('png',
+                    rf_render_png('proj_raster_0', 'proj_raster_1', 'proj_raster_2'))
+composite_df.select('png').limit(3)
```

Comment:
I might change the `3` to `2` or `4`, because we're dealing with three columns, and don't want reader to implicitly think they are related.

Reply:
right o. Was following the pattern from above chunks.